avalancheup-aws/src/main.rs

```diff
@@ -105,7 +106,10 @@ async fn main() -> io::Result<()> {
 
                 keep_resources_except_asg_ssm: sub_matches
                     .get_flag("KEEP_RESOURCES_EXCEPT_ASG_SSM"),
-                create_dev_machine: sub_matches.get_flag("CREATE_DEV_MACHINE"),
+                dev_machine: sub_matches
+                    .get_one::<Option<DevMachine>>("DEV_MACHINE")
```

Comment:
How does this work? We need to specify the JSON in the flag value?

Reply:
Hmmm yeah you're right. Maybe this doesn't work. 

Basically, we had two flags, one boolean toggle and the dev machine spec which was an `Option`. I thought I could combine those two flags down to one -- essentially it would be provide a DevMachine if you have one, or `None` in case you didn't want one. Those two flags were basically overlapping. But maybe this is the reason why you had set it up this way? 

Can they specify a DevMachine on the spec file? And avoid having to pass it in via the command-line? 

If this doesn't work I can revert that commit that removed the flag, np